main.go

```diff
@@ -245,8 +245,7 @@ func main() {
 			// Multiline interactive
 			/////////////////////
 
-			fmt.Print("\nPress Tab to submit, Ctrl + C to exit, Esc to unfocus, i to focus. When unfocused, press p to paste and c to copy response\n")
-
+			fmt.Print("\nPress Tab to submit, Ctrl + C to exit, Esc to unfocus, i to focus. When unfocused, press p to paste, c to copy response, bc to copy last code block in response\n")
```

Comment:
Why `bc`?

Reply:
I forgot to update the instructions, thanks for catching that. I was prototyping syntax like "b4c" to copy the 4th block, but discarded it due to complexity of multi-character commands.